app/components/post-meta.js

```diff
@@ -1,5 +1,18 @@
 import Ember from 'ember';
 
+/**
+  The post-meta component is a badge that presents a post's status
```

Comment:
Yeah, I don't know why the naming was that way anymore, but just further reinforces the need for the good work you're doing here documenting it all. Poorly named code can't survive in the light of good documentation!